pkg/util/operator/operators.go

```diff
@@ -0,0 +1,31 @@
+package operator
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/maistra/maistra-test-tool/pkg/util/oc"
+	"github.com/maistra/maistra-test-tool/pkg/util/pod"
+	"github.com/maistra/maistra-test-tool/pkg/util/retry"
+	"github.com/maistra/maistra-test-tool/pkg/util/shell"
+	"github.com/maistra/maistra-test-tool/pkg/util/test"
+)
+
+func GetCsvName(t test.TestHelper, operatorNamespace string, partialName string) string {
+	output := shell.Execute(t, fmt.Sprintf(`oc get csv -n %s -o custom-columns="NAME:.metadata.name" |grep %s ||true`, operatorNamespace, partialName))
+	return strings.Trim(output, "\n")
```

Comment:
`strings.TrimSpace(output)` can give the same output? Maybe worth to be more clear

Reply:
yes, you are right. I thought that that function trim only space ( not the newline symbol)